### ISSUE TITLE
Cogent "Text" parser

### ISSUE THREAD

**Issue by jarriagad** [networktocode/circuit-maintenance-parser#188]
### Environment

- circuit_maintenance_parser version: 2.2.0

A test parser for Cogent notification emails with Content-Type: text/plain

### Proposed Functionality

A text parser for cogent would provide additional coverage over incoming notifications by including emails of type `Content-Type: text/plain`

### Use Case

Not sure if this is the right way to bring this issue up. Some of my cogent notifications are not able to be parsed on their own and I believe it is due to cogent's email formatting.

Cogent email of `Content-Type: text/plain` are not able to be parsed by the current Cogent provider parsers.
Example provided below:

```
─➤  circuit-maintenance-parser --data-file "cogent_eml.eml" --data-type email --provider-type cogent                                                                  2 ↵
Provider processing failed: Failed creating Maintenance notification for Cogent.
Details:
- Processor CombinedProcessor from Cogent failed due to: 6 validation errors for Maintenance
account
  field required (type=value_error.missing)
maintenance_id
  field required (type=value_error.missing)
circuits
  field required (type=value_error.missing)
start
  field required (type=value_error.missing)
end
  field required (type=value_error.missing)
status
  field required (type=value_error.missing)
```

Example email:
[cogent_eml.eml.txt](https://github.com/networktocode/circuit-maintenance-parser/files/9945160/cogent_eml.eml.txt)

**Comment by chadell** [comment]
Thanks @jarriagad. This is right, the  Cogent Provider only support HTML parser so far, but with your example it should not be difficult to add a new parser to that provider.
To clarify, this notification looks like more for a device maintenance than a circuit. My question is about if the `Order ID(s)` are actually a Circuit ID, or some other thing.

**Comment by jarriagad** [comment]
@chadell the `Order ID(s)` are circuit IDs as far as I am aware. All cogent notifications I have are in this same style (`text/plain`) with the circuit ID listed as the `Order ID(s)`.

**Comment by chadell** [comment]
addressed in #192
